CMakeLists.txt

```diff
@@ -18,7 +18,7 @@ endif()
 
 # Check for toolchain mismatch, user might need to rebuild toolchain
 set(GCC_VERSION "13.2.0")
```

Comment:
s/what/which/ in the commit message here too.

Reply:
> Note that it comes at a cost of complicating the bootstrap process on a completely novel architecture but this hopefully won't happen often.

what's that mean?

Reply:
It means "because of stubs it became harder to compile toolchain when porting Serenity to a new architecture"